packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -544,6 +547,84 @@ Generated at ${createdAt}
     return sourceFile;
   }
 
+  /**
+   * Fill out a JSDoc heading on the SDK we're creating based on various pieces of data that may be
+   * present in the OpenAPI definition `info` object.
+   *
+   * Additionally if a license in `info.license` is recognized by the SPDX we'll also create a
+   * `LICENSE` file in for their generated SDK. We're only supporting SPDX-recognized licenses for
+   * this because we otherwise have no idea what the license represents.
+   *
+   * @see {@link https://spdx.org/licenses/}
+   */
+  private createSDKHeading() {
```

Comment:
Might be nice to also support `info.contact` but there's not really a JSDoc equivalent for that.